packages/InteractiveProfilingTool-Core.package/IPTReport.class/instance/userCallsRoots.st

```diff
@@ -0,0 +1,4 @@
+accessing
+userCallsRoots
```

Comment:
If I understand it correctly, the `userCallsRoots` will never contain a collection with more than one element. So I would prefer the name `userCallsRoot`, because it does not imply that there can be multiple roots.

Reply:
s. o. bzw. #52